.env.production

```diff
@@ -0,0 +1,9 @@
+VUE_APP_ENV=production
+VUE_APP_DOMAIN=localhost:8080
+VUE_APP_IPFS_NODE=ipfs.fleek.co
+VUE_APP_NETWORK=1
+VUE_APP_BLOCKNATIVE_DAPP_ID=032e2fb8-6c66-46a5-bf1c-a049ac7eded2
```

Comment:
The deployer will need to provide the blocknative ID.

Reply:
I'll actually rename this to .env.docker, so it doesn't interfere with our own build and deploy process.